research/gaia/gremlin/gremlin_core/src/graph_proxy/storage.rs

```diff
@@ -336,30 +292,24 @@ impl GraphProxy for DemoGraph {
     }
 }
 
-#[allow(dead_code)]
-pub fn create_demo_graph() {
-    lazy_static::initialize(&GRAPH_PROXY);
-    register_graph(GRAPH_PROXY.clone());
-}
-
 #[inline]
 fn to_runtime_vertex(
     v: LocalVertex<DefaultId>, store: &'static LargeGraphDB<DefaultId, InternalId>,
 ) -> Vertex {
     // For vertices, we query properties via vid
     let details = LazyVertexDetails::new(v.get_id(), store);
-    let id = encode_runtime_v_id(&v);
-    let label = encode_runtime_v_label(&v);
+    let id = v.get_id() as ID;
+    let label = Some(Label::Id(v.get_label()[0]));
     Vertex::new(id, label, details)
 }
 
+#[inline]
 fn to_runtime_vertex_with_property(v: LocalVertex<DefaultId>, props: &Vec<String>) -> Vertex {
```

Comment:
How can you represent no copying any property? Better take a `Option<&Vec<String>>`